Include/cpython/abstract.h

```diff
@@ -64,7 +64,7 @@ _PyVectorcall_Function(PyObject *callable)
 {
     assert(callable != NULL);
     PyTypeObject *tp = Py_TYPE(callable);
-    if (!PyType_HasFeature(tp, _Py_TPFLAGS_HAVE_VECTORCALL)) {
+    if (!Py_LIKELY(PyType_HasFeature(tp, _Py_TPFLAGS_HAVE_VECTORCALL))) {
```

Comment:
Shouldn't the entire test be inside the macro? It's possible it's fine either way, but every time I've seen __builtin_expect used (and every preexisting use of the LIKELY/UNLIKELY macros), the whole test was put inside the macro, so this would be something like: if (Py_UNLIKELY(!PyType_HasFeature(tp, _Py_TPFLAGS_HAVE_VECTORCALL))) {

Reply:
I don't think it matters. Either way would be OK. I found it easier to understand as "`_Py_TPFLAGS_HAVE_VECTORCALL` is likely" instead of "not having `_Py_TPFLAGS_HAVE_VECTORCALL` is unlikely".